input/pagecontent/terminology.md

```diff
@@ -8,23 +8,23 @@ This implementation guide does not define any unique value sets. The value sets
 
 The list below shows the value sets bound to a supported element or element slice.
 
-|ValueSet|Profile where used in|Published in|
+|ValueSet|Profile where used in|Published|
```

Comment:
For 'Profile where used in', suggest 'Profile where ValueSet used'.

Reply:
Changed to "Profile where used" to be consistent with wording choices in CodeSystem table in AU Core.